### PR TITLE
use different styling options for clear_button and clear_tag_button

### PR DESCRIPTION
**Pull request by Munksgaard**
I think this is enough.

Fixes: #75

### CODE REVIEW

**Comment by maxmarcon** [comment]
Thank you for this! I will look into it as soon as a I find some time (early next week probably)

File: test/support/helpers.ex
Comment (maxmarcon) [inline test/support/helpers.ex:1]: Let's also add:
```
    clear_button: :clear_button_extra_class,
    clear_tag_button: :clear_tag_button_extra_class,
```

to `@extend_class_option` so we test also class extension. I tested locally and some things may break, the class names need to be reordered. Try to fix it, if you're stuck, I can help

**Review by maxmarcon — CHANGES_REQUESTED** [review]
Looks great! I suggested a few changes that I think will improve the quality

File: lib/live_select.ex [hunk 1]
```diff
@@ -437,7 +437,7 @@ defmodule LiveSelect do
     doc:
       "Event to emit when the text input receives focus. The component id will be sent in the event's params"
 
-  @styling_options ~w(active_option_class available_option_class clear_button_class container_class container_extra_class dropdown_class dropdown_extra_class option_class option_extra_class text_input_class text_input_extra_class text_input_selected_class selected_option_class tag_class tag_extra_class tags_container_class tags_container_extra_class)a
+  @styling_options ~w(active_option_class available_option_class clear_button_class clear_tag_button_class container_class container_extra_class dropdown_class dropdown_extra_class option_class option_extra_class text_input_class text_input_extra_class text_input_selected_class selected_option_class tag_class tag_extra_class tags_container_class tags_container_extra_class)a
```
[the remark this answers is not in the document]
Comment (Munksgaard) [reply lib/live_select.ex]: Done in 48d24d4

File: test/support/helpers.ex
Comment (Munksgaard) [reply test/support/helpers.ex:1]: I've started this in 397a26a, but I'm still getting one test failure:

```
  1) test using both clear_button_class and clear_button_extra_class options raises (LiveSelect.ComponentTest)
     test/live_select/component_test.exs:503
     Expected exception RuntimeError but nothing was raised
     code: assert_raise(
     stacktrace:
       test/live_select/component_test.exs:504: (test)
```

Perhaps you can figure out how to fix it?

Comment (maxmarcon) [reply test/support/helpers.ex:1]: yes, I fixed it :)

**Comment by maxmarcon** [comment]
Thank you so much! 🚀

**Comment by Munksgaard** [comment]
Thank you!

**Pull request merged**